slack_formatter.py

```diff
@@ -256,11 +257,32 @@ def make_fields(self, ftext):
         fields = [{'type': 'mrkdwn', 'text': x} for x in ftext]
         return utils.chunks(fields, 10)
 
-    def reactions(self, popularity):
+    def reactions(self, popularity, count=None, denominator=1000):
+        """
+        Outputs list of blocks showing the top ten most popular reacjis.
+        Input is a dictionary with {reacji_name: count}
+        if count (int), show the number of each reacji as a frequency based on total count;
+        if denominator (int), then show it per denominator count.
+        In other words, if there were, say, 5 reacji, and count is 10000,
+        we'll show "5 :reacji: - 0.5/1000"
+        but if we provide denominator=100, for example, then we'll show
+        "5 :reacji: - .05/100"
+        """
+
+        sorted_dict = collections.OrderedDict()
+        reactions = list(popularity.keys())
+        reactions.sort(key = lambda x: popularity[x])
+        reactions.reverse()
+
         t = "*Count* *Reactji*\n"
```

Comment:
Will you please change this to:
```
Count Reactji — (#/words)
```

Reply:
Assuming you meant the header, I've added the change.